lib/matplotlib/tests/test_text.py

```diff
@@ -741,3 +741,10 @@ def test_parse_math():
     ax.text(0, 0, r"$ \wrong{math} $", parse_math=True)
     with pytest.raises(ValueError, match='Unknown symbol'):
         fig.canvas.draw()
+
+
+@image_comparison(['text_pdf_font42_kerning.pdf'], style='mpl20')
+def test_pdf_font42_kerning():
+    plt.rcParams['pdf.fonttype'] = 42
+    plt.figure()
+    plt.figtext(0.1, 0.5, "ATAVATAVATAVATAVATA", size=30)
```

Comment:
Is this the only kerning you were fixing?  If not, maybe some of the other kerning pairs would be useful here rather than repeating the same one?

Reply:
You mean pairings of glyphs? I'd hope the font defines kerning correction for more pairs. I chose this string because there the effect is obvious even without a pixel-by-pixel comparison. For this specific issue, the test targets the presence of *any* kerning. However, I will modify the string in the next iteration.

Or, are you more worried about single-byte vs multi-byte vs beyond-BMP character pairings?

Reply:
Sure, if you are sure this tests for any kerning and we don't need to test other kerning pairs, thats fine.  (I have no idea what the different pairings are technically ;-))

Reply:
The kerns come from `_text_helpers.layout` which basically just calls `font.get_kerning(prev_glyph_idx, glyph_idx, kern_mode)` repeatedly. I'm convinced by this test - if the AV spacing was bad previously and good after this change, it means that kerns are now getting inserted where they weren't previously. Multibyte characters could plausibly still be missing kerns, but if so, I think it's fine to call it a separate issue.